sdk/communication/azure-communication-identity/azure/communication/identity/_communication_identity_client.py

```diff
@@ -134,17 +140,15 @@ def delete_user(
         :return: None
         :rtype: None
         """
-        api_version = kwargs.pop("api_version", self._api_version)
         self._identity_service_client.communication_identity.delete(
             user.properties['id'],
-            api_version=api_version,
             **kwargs)
 
     @distributed_trace
     def get_token(
             self,
             user, # type: CommunicationUserIdentifier
-            scopes, # List[Union[str, CommunicationTokenScope]]
+            scopes, # type: List[Union[str, CommunicationTokenScope]]
```

Comment:
This is generating a weird change in the APIView... https://apiview.dev/Assemblies/Review/cab8292e3225490b885bc6302792ce33?diffRevisionId=16558904256e43fd87808bf0e2bfd761&diffOnly=False&revisionId=4351494b7995491cb5b0e1bcb36bacbf&doc=False#b1974ca857cd40ba899f892b7bc58fa4

just fyi, I don't think we need to take any action here